ocaml/client_records/record_util.ml

```diff
@@ -156,6 +156,28 @@ let string_to_vif_locking_mode = function
 	| "disabled" -> `disabled
 	| s -> raise (Record_failure ("Expected 'network_default', 'locked', 'unlocked', 'disabled', got "^s))
 
+let vmss_type_to_string = function
+	| `snapshot -> "snapshot"
+	| `checkpoint -> "checkpoint"
+	| `snapshot_with_quiesce -> "snapshot_with_quiesce"
+
+let string_to_vmss_type = function
+	| "snapshot" -> `snapshot
+	| "checkpoint" -> `checkpoint
+	| "snapshot_with_quiesce" -> `snapshot_with_quiesce
+	| _ -> raise (Record_failure ("Expected 'snapshot', 'checkpoint', 'snapshot_with_quiesce'"))
```

Comment:
Can you include the string which was actually received in this exception, as `string_to_network_default_locking mode` does?